packages/kbn-plugin-helpers/tasks/build/create_build.js

```diff
@@ -118,8 +123,32 @@ module.exports = function createBuild(
         options
       );
     })
+    .then(async function() {
+      if (!plugin.styleSheetToCompile) {
+        return;
+      }
+
+      const file = path.join(plugin.root, plugin.styleSheetToCompile);
+      if (!existsSync(file)) {
+        throw new Error(
+          `Path provided for styleSheetToCompile does not exist: ${file}`
+        );
+      }
+
+      const outputFileName = path.basename(file, path.extname(file)) + '.css';
+      const output = path.join(
+        buildRoot,
+        path.dirname(plugin.styleSheetToCompile),
+        outputFileName
+      );
+
+      const rendered = await sass.renderSync({ file, output });
```

Comment:
no need to `await` the sync call?

Reply:
Addressed in: https://github.com/elastic/kibana/pull/19643/commits/d9fecfefb4d3d17d96ead0791f6eab30d0288df1